### PR TITLE
test: add test for global css loader

### PR DESCRIPTION
**Pull request by haijian-vaadin**
test for https://github.com/vaadin/flow/issues/10080

### CODE REVIEW

File: vaadin-platform-hybrid-test/frontend/views/helloworldts/hello-world-ts-view.ts [hunk 1]
```diff
@@ -3,6 +3,7 @@ import '@vaadin/vaadin-button';
 import '@vaadin/vaadin-text-field';
 import { customElement, html } from 'lit-element';
 import { View } from '../view';
+import './hello-world-ts-view.global.css';
```
Comment (manolo) [inline vaadin-platform-hybrid-test/frontend/views/helloworldts/hello-world-ts-view.ts:6]: There is no assertion about whether this is working, apart from  the fact that compilation passes.
Isn't worth some code that checks that a certain element has some style applied ?

Comment (haijian-vaadin) [reply vaadin-platform-hybrid-test/frontend/views/helloworldts/hello-world-ts-view.ts:6]: agree, will do

**Comment by manolo** [comment]
The PR will not pass until a new version of flow adding the npm dependency is released

**Comment by haijian-vaadin** [comment]
> The PR will not pass until a new version of flow adding the npm dependency is released

Yes, https://github.com/vaadin/flow/pull/10085 needs to be merged and cherry-picked before this PR can turn green.

**Review by Haprog — APPROVED** [review]

**Comment by haijian-vaadin** [comment]
Flow 7.0 SNAPSHOT is currently red due to a failed test, this PR should become green once we have a green Flow 7 SNAPSHOT build.

**Pull request merged**